gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BundleTaskExtension.java

```diff
@@ -181,6 +183,16 @@ public MapProperty<String, Object> getProperties() {
 		return properties;
 	}
 
+	/**
+	 * Allow limited access to the underlying <code>instructions</code>
+	 * property so that users can control when its value is finalized.
+	 * @return View of the underlying <code>instructions</code> property.
+	 */
+	@Internal
+	public HasConfigurableValue getInstructions() {
```

Comment:
The method name leaks an implementation detail (internal field name). It also exposes 4 methods to the caller. Do we need anything more than a method which can do `instructions.disallowChanges()`?